docs/application/toc_all.md

```diff
@@ -613,6 +613,7 @@
 #### [Data Bundle](/application/native/guides/app-management/data-bundles.md)
 #### [RPC Port](/application/native/guides/app-management/rpc-port.md)
 #### [TIDL](/application/native/guides/app-management/tidl.md)
+#### [Widget Viewer](/application/native/guides/app-management/widget-viewer.md)
```

Comment:
As this new page is under guides/app-management, please check the docs/application/native/guides/app-management/overview.md file also.
It is an overview page for the sub pages under app-management,
so please add a link and a simple description for widget-viewer.md.

Reply:
Done